Symfony/CS/Tests/Fixer/PSR2/FunctionCallSpaceFixerTest.php

```diff
@@ -80,6 +80,15 @@ public function testFixProvider()
             array(
                 '<?php $a = 2 * (1 + 1);',
             ),
+            array(
+                '<?php echo ($a == $b) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php echo ($a == test($b)) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php include ($html)? "custom.html": "custom.php";',
```

Comment:
@gharlan thanks.
First of all i need to finish this PR and then open new issue with custom function call like `$foo()`